c++/test/TestDecompression.cc

```diff
@@ -669,6 +669,11 @@ namespace orc {
       buf[2] = static_cast<char>(compressedSize >> 15);
     }
 
+    void writeUnCompressedHeader(size_t compressedSize) {
```

Comment:
According to the Apache ORC convention, could you rename `writeUnCompressedHeader` to `writeUncompressedHeader` ?